### PR TITLE
#169257386 Integrate HoundCI with readme badge

### PR DESCRIPTION
**Pull request by gildniy**
#### What does this PR do?
This pull request sets up Integrates HoundCI with the readme badge for automated code review

#### Description of Task to be completed
- [x] Connect the project to [Hound CI website](https://houndci.com/)
- [x] Place the Hound CI badge
- [x] Choose a `.jshintrc` file version in the Dashboard for setting and allow ECMAScript 6 features such as `const`

#### How should this be manually tested?
- At the point of raising a Pull Request, the Hound will review the commit and a Hound CI logo will appear below the PR description
- On the README a badge will appear

#### Any background context you want to provide?
This is to automatically validate the codebase with standard style guides

#### What are the relevant pivotal tracker stories?
[#169257386](https://www.pivotaltracker.com/story/show/169257386)

#### Screenshots (if appropriate)
image

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -92,4 +92,10 @@ typings/
 # DynamoDB Local files
 .dynamodb/
 
+# Ide files
+.idea
+
+# Lock files
+/package-lock.json
```
Comment (b0nbon1) [inline .gitignore]: Nice but why add package-lock.json to the .gitignore file?

**Review by b0nbon1 — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (gildniy) [reply .gitignore]: Because it autogenerates upon dependencies install, along with `node_modules` folder, so no need to push it.

Comment (timolinn) [reply .gitignore]: According to npm it should be checked into source control. https://github.com/npm/npm/blob/v5.0.0/doc/files/package-lock.json.md

Comment (gildniy) [reply .gitignore]: Let me remove it from .gitignore I saw, it was not meant to be ignored. I was a mistake.

**Review by timolinn — APPROVED** [review]

**Comment by timolinn** [comment]
@nshaphan kindly review

**Review by timolinn — DISMISSED** [review]

**Review by nshaphan — DISMISSED** [review]

**Comment by timolinn** [comment]
@b0nbon1 kindly review

**Review by b0nbon1 — DISMISSED** [review]

**Comment by b0nbon1** [comment]
Everything is okay, Please resolve the conflicts before merging

**Comment by timolinn** [comment]
@gildniy Kindly resolve conflicts

**Comment by gildniy** [comment]
> @gildniy Kindly resolve conflicts

Fixed

**Review by b0nbon1 — DISMISSED** [review]
It now looks great

**Review by nshaphan — DISMISSED** [review]
great work

**Review by b0nbon1 — DISMISSED** [review]

**Review by timolinn — APPROVED** [review]

**Review by nshaphan — APPROVED** [review]

**Review by b0nbon1 — APPROVED** [review]

**Pull request merged**